lib/fog/serverlove.rb

```diff
@@ -1,10 +1,10 @@
-require 'fog/serverlove/version'
 require 'fog/core'
 require 'fog/json'
+require File.expand_path('../serverlove/version', __FILE__)
 
 module Fog
   module Compute
-    autoload :Serverlove, 'fog/compute/serverlove'
+    autoload :Serverlove, File.expand_path('../compute/serverlove', __FILE__)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.